lib/rialto/etl/cli/grants.rb

```diff
@@ -105,6 +105,8 @@ def extract(row, profile_id, force)
             return extract_file
           end
 
+          return extract_file unless results.any? # this will avoid creating an emtpy file
```

Comment:
I have a minor preference for using `if` over `unless`, assuming it doesn't make the code less readable. In this case, that'd be:

```ruby
	return extract_file if results.empty?
```

What would you think about that?

Reply:
Good comment, too. Can you `s/emtpy/empty/`?